ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -153,6 +153,8 @@ message DamlTransactionEntry {
 
   // The nonce used to generate contract ids
   bytes submission_seed = 5;
+
+  google.protobuf.Timestamp submission_time = 6;
```

Comment:
comment would be helpful: what is it and what it's used for.

Reply:
done